src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -21,6 +21,7 @@
 DOCKER_IMAGE_UP_TO_DATE = "Image is up to date for {}"
 IMAGE = "mcr.microsoft.com/mcr/hello-world:latest"
 FAQ_MESSAGE = "\nPlease refer to https://aka.ms/acr/health-check for more information."
+ERROR_MSG_DEEP_LINK = "\nPlease refer to https://aka.ms/acr/errors#{} for more information."
```

Comment:
Can we combine FAQ_MESSAGE and ERROR_MSG_DEEP_LINK

Reply:
The `FAQ_MSG` and `ERROR_MSG` point to different URLs. 
FAQ_MSG is used only in one [place](https://github.com/Azure/azure-cli/blob/9601cf2b5678ce7056c6f4d2f3cf8ae3b0ddf07f/src/azure-cli/azure/cli/command_modules/acr/check_health.py#L294).